src/frontend/Felles/PersonHeader/BehandlingTags.tsx

```diff
@@ -0,0 +1,74 @@
+import { Behandling, BehandlingKategori, kategoriTilTekst } from '../../App/typer/fagsak';
+import { Tag } from '@navikt/ds-react';
+import { stønadstypeTilTekst, stønadstypeTilTekstKort } from '../../App/typer/behandlingstema';
+import {
+    behandlingstypeTilTekst,
+    behandlingstypeTilTekstKort,
+} from '../../App/typer/behandlingstype';
+import { Behandlingsårsak, behandlingsårsakTilTekst } from '../../App/typer/Behandlingsårsak';
+import React from 'react';
+import styled from 'styled-components';
+
+const TagLitenSkjerm = styled(Tag)`
+    @media screen and (min-width: 1190px) {
+        display: none;
+    }
+
+    @media screen and (max-width: 805px) {
+        display: none;
+    }
+`;
+
+const TagStorSkjerm = styled(Tag)`
+    @media screen and (max-width: 1189px) {
+        display: none;
+    }
+`;
+
+interface Props {
+    behandling?: Behandling;
+}
+
+const BehandlingTags: React.FC<Props> = ({ behandling }) => {
+    return (
```

Comment:
Siden alt her "krever" at behandling ikke er null - kan vi ikke kalle på denne hvis behandlingen er null? Og heller sørge for at den sjekken skjer utenfor?

Reply:
Forsåvidt enig i det, siden alt krever behandling uansett

Reply:
[95ec79b](https://github.com/navikt/familie-ef-sak-frontend/pull/2267/commits/95ec79bd39e168a59e697312e3ec37dac931927d)